actions/channel_actions.jsx

```diff
@@ -79,9 +79,9 @@ export function loadChannelsForCurrentUser() {
         for (const id of unreads) {
             const channel = getChannel(state, id);
             if (channel && channel.type === Constants.DM_CHANNEL) {
-                loadNewDMIfNeeded(channel.id);
+                dispatch(loadNewDMIfNeeded(channel.id));
```

Comment:
`loadNew(DM|GM)IfNeeded` is an action that must be dispatched -- calling it plain just returned an unused async function.

Reply:
I wonder if this is something we could add an ESLint check for at some point. You'd have to find a function call that returns a redux action and then ensure that something is done with it. I guess that requires ESLint to understand types though, so that would probably be difficult.